onnxruntime/core/framework/execution_frame.cc

```diff
@@ -73,6 +73,13 @@ Status IExecutionFrame::GetOrCreateNodeOutputMLValue(int index, const TensorShap
   return status;
 }
 
+bool IExecutionFrame::TryGetInferredShape(int /*index*/, TensorShape& /*shape*/) const {
```

Comment:
>IExecutionFrame::TryGetInferredShape [](start = 5, length = 36)

Out of curiosity, do we ever expect this to be called or this function should always be overrided by derived class? If it is not legit to call directly on this method, maybe we can put an error message or ORT_NOT_IMPLEMENTED to indicate that instead of failed silently?